src/routes/_surfaces/inspector/Model.svelte

```diff
@@ -340,34 +353,73 @@
         }}
       >
         <MenuItem
-          on:select={() => {
+          on:select={async () => {
```

Comment:
Can you move this event handler outside of html? Add it to the script tag and call it from here. Same with event handler below.